docs/developers-guide.md

```diff
@@ -55,6 +55,12 @@ The prevalue editor allows you to configure the following properties.
 
 Once your data type has been configured, simply set-up a property on your page doc type using your new data type and you are set to start editing.
 
+#### Name Template
+
+You can choose the template for generating list item labels using the syntax `{{propertyAlias}}`. If you would like to include the index position in the label, you can use `{{$index}}`.
+
+If your property links to a content node, you can use the Angular filter `{{ pickerAlias | ncNodeName }}` to show the node name rather than the node ID. If it's linking to a media node, you can use `{{ pickerAlias | ncMediaName }}`. These features were added in 0.4.0.
```

Comment:
If you remove the part about `ncMediaName`, I will merge it in 👍

Reply:
I struggled with editing this pull request and managed to create a new pull request on my own repo. 👎 

I've created a new pr: https://github.com/umco/umbraco-nested-content/pull/146